tests/modules/programs/gh/extensions.nix

```diff
@@ -0,0 +1,18 @@
+{ config, lib, pkgs, ... }:
+
+{
+  config = {
+    programs.gh = {
+      enable = true;
+      extensions = [ pkgs.gh-eco ];
```

Comment:
Need to add a stub for `gh-eco` as well to avoid the unnecessary download. Adding

```
test.stubs.gh-eco = {
  buildScript = "mkdir -p $out/bin; touch $out/bin/gh-eco";
};
```

may suffice.

Reply:
Thanks, that makes sense. I had to slightly expand this, in particular, I changed `tests.stubs` to set `pname` in a9ca816d5df484f29d69196de7f85d8688a8a9fd.